docs/ref/models/constraints.txt

```diff
@@ -299,9 +302,11 @@ The error message used when ``ValidationError`` is raised during
 :ref:`model validation <validating-objects>`. Defaults to
 :attr:`.BaseConstraint.violation_error_message`.
 
-This message is *not used* for :class:`UniqueConstraint`\s with
+.. versionadded:: 5.1
+
+This message *is* also used for :class:`UniqueConstraint`\s with
 :attr:`~UniqueConstraint.fields` and without a
-:attr:`~UniqueConstraint.condition`. Such :class:`~UniqueConstraint`\s show the
-same message as constraints defined with
-:attr:`.Field.unique` or in
-:attr:`Meta.unique_together <django.db.models.Options.constraints>`.
+:attr:`~UniqueConstraint.condition`. The error message of such 
+:class:`~UniqueConstraint`\s will then be different from constraints defined 
+with :attr:`.Field.unique` or in :attr:`Meta.unique_together 
+<django.db.models.Options.constraints>`.
```

Comment:
To me it looks like `unique_together` is linked to the wrong place, I think it should link to [`unique_together`](https://docs.djangoproject.com/en/5.0/ref/models/options/#unique-together)

Reply:
> To me it looks like unique_together is linked to the wrong place, I think it should link to [unique_together](https://docs.djangoproject.com/en/5.0/ref/models/options/#unique-together)

This is not addressed and this would also be the case above